src/pages/crm-reviews-page/ReviewsList/ReviewsListItem/ReviewsListItem.tsx

```diff
@@ -0,0 +1,128 @@
+import styles from './reviews-list-item.module.scss'
+import IconPlus from '../../../../icons/plus.svg?react'
+import { ReviewsRating } from '../ReviewsRating/ReviewsRating'
+import IconButton from '@mui/material/IconButton'
+import DeleteIcon from '../../../../icons/delete.svg?react'
+import Tooltip from '@mui/material/Tooltip'
+import ChangeIcon from '../../../../icons/crm-file.svg?react'
+import { Images } from '../../../../redux/feedbacks/slice'
+import { ModalCustom } from '../../../../components/Modal-custom-btn/ModalCustomWindow'
+import { useState } from 'react'
+import axiosInstance from '../../../../axios/settings'
+
+export interface ReviewsListItemProps {
+  id: number
+  rating: number
+  name: string
+  time: string
+  description: string
+  isActive: boolean
+  images: Images[]
+  onStatusChange: () => void
+}
+
+export const ReviewsListItem = ({
+  id,
+  rating,
+  name,
+  time,
+  description,
+  isActive,
+  images,
+  onStatusChange
+}: ReviewsListItemProps) => {
+  const [openModal, setOpenModal] = useState(false)
+
+  const handleOpenModal = () => {
+    setOpenModal(true)
+  }
+
+  const isDeleteDisabled = !isActive
+  const date = new Date(time)
+  const formattedDate = date.toLocaleDateString('uk-UA', {
+    day: '2-digit',
+    month: '2-digit',
+    year: 'numeric'
+  })
+
+  const handleAddToArchive = () => {
+    const fetchAdd = async () => {
+      try {
+        await axiosInstance.put('/api/reviews/archive', { id })
+
+        onStatusChange()
+      } catch (err) {
+        console.error(err)
+      }
+    }
+    fetchAdd()
+  }
+
+  const handleRemoveFromArchive = () => {
+    const fetchRemove = async () => {
+      try {
+        await axiosInstance.put('/api/reviews/unarchive', { id })
+
+        onStatusChange()
+      } catch (err) {
+        console.error(err)
+      }
+    }
+    fetchRemove()
+  }
+  return (
+    <li className={styles.item}>
```

Comment:
if i am not mistaken, it is just flex list of items, so, why do you use `li`?

Reply:
Should I use 'div' or something from mui?

Reply:
yeah, just div
use lists tags only when you have actual list